graphql/schema.graphql

```diff
@@ -160,6 +160,10 @@ type User {
   username: String!
   displayName: String!
   email: String!
+  apiKey: String!
+  accessToken: String!
+  refreshToken: String!
+  roles: [String!]!
```

Comment:
do you know what purpose these fields will serve on the client? i'm not aware of any. for a lot of these i feel like they are determined on the back end from context. not totally sure we need them exposed on the client but maybe you know something i dont?

Reply:
I just removed fields that aren't currently consumed by Spruce

Reply:
I just removed fields that aren't currently consumed by Spruce